lib/vm/src/compatability.rs

```diff
@@ -1,21 +1,14 @@
-use wasm_nm::{Options, Symbol, Symbols};
+use parity_wasm::elements::{Deserialize, Module};
 
 use crate::errors::{Result, ValidationErr};
 
-static PUBLIC_SYMBOLS: Options = Options {
-    imports: true,
-    exports: true,
-    privates: false,
-    sizes: false,
-};
-
 /// Lists all imports we provide upon instantiating the instance in Instance::from_module()
 /// This should be updated when new imports are added
 static SUPPORTED_IMPORTS: &[&str] = &[
-    "read_db",
-    "write_db",
-    "canonicalize_address",
-    "humanize_address",
+    "env.read_db",
```

Comment:
interesting that namespaces are preserved here

Reply:
I compose the strings manually in https://github.com/confio/cosmwasm/pull/134/files#diff-6537b402b8ff635cc4a489e46f9dabf7R57. Wasm seems to import always into one specific module. In AssemblyScript I control this by the .ts module. In Rust, the C interface seem to be imported into the module `env` by default.

So I can write an AssemblyScript contract that requires `foobar.read_db` instead of `env.read_db`.